Dockerfile

```diff
@@ -1,12 +1,13 @@
-# The base go-image
-FROM golang:1.21-alpine
+# Builder image
+FROM golang:1.21-alpine as build
 
 RUN mkdir /app 
 COPY . /app
-
 WORKDIR /app
+RUN go build
 
-RUN go build -v
+# Server image
+FROM gcr.io/distroless/static-debian12
 
-# Run the server executable
+COPY --from=build /app/boyd_cooper /app/boyd_cooper
 CMD [ "/app/boyd_cooper" ]
```

Comment:
There is a small issue with deployment. Either the docker compose file needs to be edited to place `quotes.txt` into in `/`. Or you can add `WORKDIR /app` to this Dockerfile.

Reply:
Ah, excellent point! I will do this